tests/pipelines/unclip/test_unclip_image_variation.py

```diff
@@ -421,7 +421,12 @@ class DummyScheduler:
     def test_attention_slicing_forward_pass(self):
         test_max_difference = torch_device == "cpu"
 
-        self._test_attention_slicing_forward_pass(test_max_difference=test_max_difference)
+        # Check is relaxed because there is not a torch 2.0 sliced attention added kv processor
+        expected_max_diff = 1e-2
+
+        self._test_attention_slicing_forward_pass(
+            test_max_difference=test_max_difference, expected_max_diff=expected_max_diff
+        )
```

Comment:
Should this be conditioned on the torch version being used?

Reply:
Sorry, intention is to follow up and fix the test data